docs/platform/tutorials/quick-start/quick-start.md

```diff
@@ -30,7 +30,7 @@ If you need help with this guide, then please join our public Slack community [`
 
 ## Part 1. Deploy a chat UI with sentiment analysis
 
-To use Quix effectively in the shortest possible time, you will initially use prebuilt samples from the [Quix Samples](https://github.com/quixio/quix-samples){target=_blank}. These open source samples have already been coded and tested by Quix engineers, and other contributors. All you have to do is configure them (if required) and deploy them to your workspace.
+To use Quix effectively in the shortest possible time, you will initially use prebuilt Code Samples from the [Code samples](https://github.com/quixio/quix-samples){target=_blank}. These open source Code Samples have already been coded and tested by Quix engineers, and other contributors. All you have to do is configure them (if required) and deploy them to your workspace.
```

Comment:
> use prebuilt Code Samples from the [Code samples](h

Perhaps:

use prebuilt Code Samples from their [repository](?

Reply:
that doesn't make sense to me. but done.